src/components/ImageDetails/VPhotoDetails.vue

```diff
@@ -29,23 +29,29 @@
         {{ $t('photo-details.legal-disclaimer') }}
       </p>
 
-      <VButton
-        variant="action-button"
-        size="disabled"
-        class="report py-2 mt-2"
-        @click="reportForm.toggleVisibility"
-      >
-        {{ $t('photo-details.content-report.title') }}
-        <VIcon :icon-path="icons.flag" class="text-trans-blue ms-2 text-sm" />
-      </VButton>
-      <VContentReportForm
-        v-if="isReportFormVisible"
-        :image="image"
-        :provider-name="providerName"
-        data-testid="content-report-form"
-        class="mt-2 text-left"
-        @close-form="reportForm.close()"
-      />
+      <VPopover z-pos="20">
+        <template #trigger>
+          <VButton
+            variant="action-button"
+            size="disabled"
+            class="report py-2 mt-2"
```

Comment:
This isn't an actual variant of the button. Because it's a link-style button should it be the `plain` variant? Also, setting the size to disabled causes the `x` axis margins to be too tight. The padding being applied in the `class` prop matches the `medium` (default) size anyway, so we can just remove the size prop altogether.

Finally, there are some accessibility props that get passed to the trigger.

```suggestion
        <template #trigger="{ a11yProps }">
          <VButton
            v-bind="a11yProps"
            variant="plain"
            class="mt-2"
```

Reply:
Committed in dfb74bd.